test/common/tracing/http_tracer_impl_test.cc

```diff
@@ -442,24 +442,20 @@ class HttpTracerImplTest : public Test {
 };
 
 TEST_F(HttpTracerImplTest, BasicFunctionalityNullSpan) {
-  EXPECT_CALL(config_, operationName()).Times(2);
+  EXPECT_CALL(config_, operationName()).Times(1);
```

Comment:
Curious why this changes..

Reply:
In `HttpTracerImpl::startSpan` I removed the initialisation of the span name based on the `operationName()`, as it was not being used in the ZipkinTracer (it was overriding the span name to use the host). I need to check with someone from LightStep to see if it has an impact on their tracer - from initial inspection it seems like the span name is just passed through to the tracer, so I'm guessing Igress/Egress would just appear as the span names in their UI. However this is not that meaningful, as the span.kind can be used to indicate direction.

However this is something that needs to be checked before this PR is merged.

Reply:
Any update on this? PR looks LGTM otherwise.

Reply:
I've been in contact with a couple of people at LightStep who said they would try to look at it, but not sure when.

Reply:
cc @bhs @tedsuo @rnburn Any thoughts appreciated - would ideally like to get this PR merged, so just need to understand if a change in the operation name provided to the LightStep tracer would have any impact on your users?

Reply:
Per other comment and per @RomanDzhabarov yes it effects the UI. I'm not sure why we need to change the default behavior. If no operation name is specified, do what we do today. If operation name is specified, override it. If tracer wants to override that, no problem.

Also nit: Times(1) is redundant.